engines/stark/ui/menu/mainmenu.cpp

```diff
@@ -181,8 +172,24 @@ void MainMenuScreen::settingsHandler() {
 	StarkUserInterface->changeScreen(Screen::kScreenSettingsMenu);
 }
 
+void MainMenuScreen::boxHandler() {
+	if (!StarkSettings->isDemo() && StarkSettings->hasBookOfSecrets()) {
+		StarkUserInterface->changeScreen(kScreenGame);
+		StarkResourceProvider->initGlobal();
+		StarkScene->setFadeLevel(1.0f);
+		StarkResourceProvider->requestLocationChange(0x7c, 0x00);
+	}
+}
+
 void MainMenuScreen::quitHandler() {
 	StarkUserInterface->notifyShouldExit();
 }
 
-} // End of namespace Stark
\ No newline at end of file
+VersionInfoText::VersionInfoText() :
+		StaticLocationWidget("VERSION INFO REALLY", nullptr, nullptr) {
+	Common::String text = _copyrightSymbol + Common::String("1999 Funcom");
+	_renderEntry->getText()->setText(text);
+	_renderEntry->setPosition(Common::Point(_posX, _posY));
```

Comment:
The default positions of the "VERSION INFO HERE" widget in the released game and the demo are different, and both of them needs to be moved, so I chose to hard-code it.